wolox-books/src/scss/equalizer.scss

```diff
@@ -0,0 +1,92 @@
+$equalizer-black: #000;
```

Comment:
It's better to install https://www.npmjs.com/package/wolox-equalizer so there's no need to actually copypasting this file on each project 😉